container/README.md

```diff
@@ -13,20 +13,141 @@ If a section or action is tagged as "Fedora", it should be applied only for a
 Fedora deployment. If the tag "docker-compose" is used it will be used for the
 docker-compose deployment.
 
-# Get container images
+## Run the openQA in HA mode (docker-compose)
+
+To create the containers. In the directory openQA/docker/webui, execute:
+
+    docker-compose up -d
+
+## How to execute a job
+
+To access to the scripts to launch jobs the webui container can be used
+
+    docker-compose webui /usr/share/openqa/script/clone_job.pl
+
+Using the default configuration (Fake auth) an api/key pair is automatically created
+
+    1234567890ABCDEF / 1234567890ABCDEF
+
+But before launching a JOB is mandatory to add the tests for this job in the directory workdir/tests
+
+For instance, Launch an opensuse test you can follow these steps
+
+```
+cd workdir/tests
+git clone git@github.com:os-autoinst/os-autoinst-distri-opensuse.git
+ln -s os-autoinst-distri-opensuse opensuse
+git clone https://github.com/os-autoinst/os-autoinst-needles-opensuse.git
+mv os-autoinst-needles-opensuse/ opensuse/products/opensuse/needles
+```
+
+NOTE: To execute opensuse test the worker needs some dependencies, install them as follow:
+    
+    docker-compose exec worker zypper in -y os-autoinst-distri-opensuse-deps
+
+To launch a job using clone_job, execute: 
+
+    docker-compose webui /usr/share/openqa/script/clone_job.pl --apikey 1234567890ABCDEF --apisecret 1234567890ABCDEF --host nginx https://openqa.opensuse.org/tests/1613965
+
+
+### Change the number web UI replicas (optional)
+
+To set the number of replicas set the environment variable
+`OPENQA_WEBUI_REPLICAS` to the desired number. If this is not set, then the
+default value is 2.
+
+    export OPENQA_WEBUI_REPLICAS=3
+
+Or, you can set the variable in-line 
+    
+    OPENQA_WEBUI_REPLICAS=3 docker-compose up -d
+
+Additionally you can edit the .env file to set the default value for this
+variable.
+
+### Change the container images (optional)
+
+By default the the docker-compose will use the images of openqa_webui, 
+openqa_webui_lb and openqa_worker stored in registry.opensuse.org.
+
+The selection of other images can be done setting up the environment variables: 
+OPENQA_WEBUI_IMAGE, OPENQA_WEBUI_LB_IMAGE, and OPENQA_WORKER_IMAGE
+
+For instance, to build and use local images follow these steps
+
+    $ docker-compose build
+    $ OPENQA_WEBUI_IMAGE=openqa_webui OPENQA_WEBUI_LB_IMAGE=openqa_webui_lb OPENQA_WORKER_IMAGE=openqa_worker docker-compose up -d
+
+### Add the exported port for the load balancer (optional)
+
+By default the load balancer exposes the web UI on ports 9526, 80 and 443.
+
+```
+ports:
+  - "9526:9526"
+  - "80:9526"
+```
+
+### Enable the SSL access to the load balancer (optional)
+
+Enable the SSL access in three steps:
+
+1. To expose the SSL port uncomment this line in the docker-compose.yaml file
+   in the service nginx.
+
+```
```

Comment:
It would be nice to fix the indentation so the additional paragraphs within the list items actually belong to them.